doc/external-libs/lapack.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+	  "http://www.w3.org/TR/html4/loose.dtd">
+
+<html>
+  <head>
+    <title>The deal.II Readme on interfacing to BLAS/LAPACK</title>
+    <link href="../screen.css" rel="StyleSheet">
+    <meta name="copyright" content="Copyright (C) 2010 - 2017 by the deal.II authors">
+    <meta name="keywords" content="deal.II">
+  </head>
+  <body>
+
+    <h2>Installation of <acronym>BLAS/LAPACK</acronym></h2>
+
+    <p>
+      <a href="https://netlib.org/lapack/">LAPACK</a>
+      is a library of Fortran subroutines for solving the most
+      commonly occurring problems in numerical linear algebra.
+      LAPACK is available on <a href="https://github.com/Reference-LAPACK/lapack">GitHub</a>. LAPACK releases are also available on <a href="https://netlib.org/lapack/">netlib</a>.
+    </p>
+
+    <p>
+      Below is a short summary of instructions on how to compile and install
+      <acronym>LAPACK</acronym> by hand (for the case you wish to do so).
```

Comment:
First choice would be to use it from OS sources (like ``apt install libblas* liblapack*``).

Reply:
Hi Prof. Heister,
I have added the same in the document.